cmd/openshift-install/create.go

```diff
@@ -897,7 +898,12 @@ func handleUnreachableAPIServer(ctx context.Context, config *rest.Config) error
 		return fmt.Errorf("failed to fetch %s: %w", lbConfig.Name(), err)
 	}
 
-	_, ipAddrs, err := lbConfig.ParseDNSDataFromConfig(lbconfig.PublicLoadBalancer)
+	lbType := lbconfig.PublicLoadBalancer
+	if installConfig.Config.Publish == types.InternalPublishingStrategy {
```

Comment:
Should we use func `PublicAPI` from the installconfig? It is just a unified way to check but current way should be OK since it looks like only azure supports it.

https://github.com/openshift/installer/blob/47bdc5351db43b3528c382b5749ab926fe8bf324/pkg/types/installconfig.go#L657-L666

Reply:
I hadn't seen PublicAPI() before. Nice catch @tthvo